### ISSUE TITLE
using umatrix for certain chrome-scheme's where requests to many websites are made

### ISSUE THREAD

**Issue by berrythesoftwarecodeprogrammar** [gorhill/uMatrix#359]
i use 2 addons for keeping up with websites:
brief (rss reader) https://addons.mozilla.org/en-us/firefox/addon/brief/
update scanner (checks webpages for changes) https://addons.mozilla.org/en-us/firefox/addon/update-scanner/

the urls they use are:
chrome://brief/content/brief.xul
and
chrome://updatescan/content/diffPage.xul?*
respectively

since they are chrome-scheme's, all requests made on them are allowed by default of course. however ive turned matrix off for 'brief.chrome-scheme' and 'updatescan.chrome-scheme' and allowed them to make css and image requests to *. i feel thats better than them allowing *. and i think this way, blocked domains will now be blocked on those schemes?

i guess i was going to ask if it were at all possible to let umatrix know which domain was being accessed on either brief or update scanner so that umatrix could be used the same way while in those schemes. i think its probably not possible though without affecting performance and doing weird things. updatescanner pages look like this:
chrome://updatescan/content/diffPage.xul?id=1657&title=Borderlands%202%3A%20Golden%20Key%20-%20Orcz.com%2C%20The%20Video%20Games%20Wiki&url=http%3A//orcz.com/Borderlands_2%3A_Golden_Key&oldDate=11%20weeks%20ago&newDate=today%20at%2016%3A57&delay=0
so the url could be fetched from there and unescaped but i know this isnt the job of umatrix and there probably isnt another way to find out the domain being accessed by updatescan in the current tab. and brief never has the domains in the url. it stays as 'chrome://brief/content/brief.xul', so that bad method is ruled out anyway

but the point is, both of these addons, and maybe all other addons such as rss readers, basically let you browse web pages, but because of the scheme they use, its hard to use umatrix on them the same way as normal websites. it would be nice if for example on brief when im reading a feed, it would know which website's feed im reading and block third party requests for that site as normal, but i dont want to do it for all sites because i dont want to break all other feeds. i want the same functionality that umatrix has on normal web browsing. but i understand if its completely out of the question because its impossible or something

tl;dr wondering if possible to make umatrix function like normal on chrome-schemes of addons like rss readers so that the same rules can be used on them even when the domain/url isnt readily available

**Comment by Atavic** [comment]
.rss files are nothing more than improved xml files with a title, a short intro and a link to the complete article. Look into the code of brief (rss reader) if there's something objectionable.
